doc/blog/Wasm-JS series/Blog - WASM.md

```diff
@@ -0,0 +1,672 @@
+
+That aslo means you can control how this element is created.  By default is a `<div>`, but you can change this directly in the constructor by setting the `htmlTag` parameter to whatever required. Example:
+
+``` csharp
+// MyControl constructors
+public MyControl() : base() // will create a "div" HTML element
+public MyControl() : base("input") // Will create an "input" HTML element
+public MyControl() : base(htmlTag: "span") // Will create a "span" HTML element
+```
+
+Once created, it's possible to interact directly with this element by calling helper methods supplied by Uno on base classes. Obviously those methods are only available when targeting the _Wasm_ platform.
+
+Helper methods:
+
+* The method `base.SetStyle()` can be used to set a CSS Style on the html element. Example:
+
+  ``` csharp
+  // Setting only one CSS style
+  SetStyle("text-shadow", "2px 2px red");
+  
+  // Setting many CSS styles at once using C# tuples
+  SetStyle(("text-shadow", "2px 2px blue"), ("color", "var(--app-bg-color)"));
+  ```
+
+* The method `base.ResetStyle()` can be used to set CSS styles to their default values. Example:
+
+  ``` csharp
+  // Reset text-shadow style to its default value
+  ResetStyle("text-shadow");
+  
+  // Reset both text-shadow and color to their default values
+  ResetStyle("text-shadow", "color");
+  ```
+
+* The methods `base.SetAttribute()` and `base.RemoteAttribute()` can be used to set HTML Attributes on the html element. Example:
+
+  ``` csharp
+  // Set the "href" attribute of an <a> element
+  SetAttribute("href", "#section2");
+  
+  // Set many attributes at once
+  SetAttribute(("target", "_blank"), ("referrerpolicy", "no-referrer"));
+  
+  // Remove attribute from DOM element
+  RemoveAttribute("href");
+  ```
+
+* The method `base.SetHtmlContent()` could be used to set arbitrary HTML content as child of the control.
+
+  ``` csharp
+  SetHtmlContent("<h2>Welcome on Uno Platform!</h2>");
+  ```
+
+  > Note: don't use this unless your control doesn't include any _managed_ children, or you'll get unexpected results.
+
+* Finally, it's possible to invoke an arbitrary bunch of JavaScript code by using the _static_ method `WebAssembleRuntime.InvokeJS()`. Once there, it directly in the browser, so it's possible to make almost anything. It's possible to use the `HtmlId` property of the element to locate it in JavaScript code.
+  If the control has been loaded (after de `Loaded` router event), it will be available directly by calling the JavaScript `document.getElementById()`. But it's also possible to access it before using the  `Uno.UI.WindowManager.current.getView(<HtmlId>)` function in JavaScript.
```

Comment:
```suggestion
  If the control has been loaded (after de `Loaded` routed event), it will be available immediately by calling the JavaScript `document.getElementById()`. But it is also possible to access it before using the  `Uno.UI.WindowManager.current.getView(<HtmlId>)` function in JavaScript.
```